src/Components/Nav/InnerNav/InnerNav.js

```diff
@@ -0,0 +1,68 @@
+/*eslint-disable*/
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './InnerNav.scss';
+
+class InnerNav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      isMenuHover: false,
+    };
+  }
+  toggleHover = () => {
+    this.setState({
+      isMenuHover: !this.state.isMenuHover,
+    });
+  };
+
+  render() {
+    const { isMenuHover } = this.state;
+    const { name, link, cate, depth, banner } = this.props;
+    return (
+      <li onMouseEnter={this.toggleHover} onMouseLeave={this.toggleHover}>
+        <Link to={link}>{name}</Link>
+        <div className={`navSubWrap  ${isMenuHover ? 'active' : ''}`}>
+          <div className="innerWrap">
+            <div className="specialProduct">
+              <ul className="lists">
+                {cate.map(item => {
```

Comment:
```suggestion
                {cate.map(item => {
                const {id, name, link} = item;
```
item 구조분해할당해서 사용해도 좋겠네요